latest 1.2.0 release fails to build with "format not a string literal"

Hi,

I tried on Mageia Linux Cauldron to build latest 1.2.0 release but it fails to build with an error  "format not a string literal":

> /home/david/mgarepo/hydrogen/BUILD/hydrogen-1.2.0/src/core/Logger.cpp:58:32: error: format not a string literal and no format arguments [-Werror=format-security]
   58 |                         fprintf( stderr,
      |                         ~~~~~~~^~~~~~~~~
   59 |                                          QString( "Error: can't open log file [%1] for writing...\n" )
      |                                          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   60 |                                          .arg( logger->m_sLogFilePath ).toLocal8Bit().data() );
      |                                          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
cc1plus: some warnings being treated as errors
gmake[2]: *** [src/core/CMakeFiles/hydrogen-core-1.2.0.dir/build.make:737: src/core/CMakeFiles/hydrogen-core-1.2.0.dir/Logger.cpp.o] Error 1
gmake[2]: *** Waiting for unfinished jobs....


Attached full build log
[log.hydrogen.txt](https://github.com/hydrogen-music/hydrogen/files/11183608/log.hydrogen.txt)

--- a/src/core/Logger.cpp
+++ b/src/core/Logger.cpp
@@ -55,7 +55,7 @@ void* loggerThread_func( void* param ) {
 	if ( logger->__use_file ) {
 		log_file = fopen( logger->m_sLogFilePath.toLocal8Bit().data(), "w" );
 		if ( ! log_file ) {
-			fprintf( stderr,
+			fprintf( stderr, "%s",
 					 QString( "Error: can't open log file [%1] for writing...\n" )
 					 .arg( logger->m_sLogFilePath ).toLocal8Bit().data() );
 		}

